### PR TITLE
Allow web tests to fail

### PR DESCRIPTION
**Pull request by jonahwilliams**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ task:
         cpu: 4
         memory: 12G
     - name: web_tests-linux
+      allow_failures: true
```
Comment (goderbauer) [inline .cirrus.yml]: Maybe add a TODO to remove this ones these are stable.

**Review by goderbauer — APPROVED** [review]
LGTM

File: .cirrus.yml (hunk 1 above)
Comment (jonahwilliams) [reply .cirrus.yml]: Done

**Review by tvolkert — APPROVED** [review]
LGTM

**Comment by jonahwilliams** [comment]
Landing on red to reduce flakiness of build

**Pull request merged**